packages/fiori/src/themes/ShellBar.css

```diff
@@ -97,7 +97,6 @@ slot[name="profile"] {
 	width: 2rem;
 	height: 2rem;
```

Comment:
I noticed that these are the measures for the XS avatar. Isn't it a good idea to add the font-size too? It looks strange when we have avatar with measurement of XS avatar and font-size of S avatar.
- S:
![image](https://github.com/user-attachments/assets/81f075b5-4e0d-4f52-b9d1-41267019d82e)
- XS:
![image](https://github.com/user-attachments/assets/a4c9d3f9-1b1e-42f5-aa01-aae752e5313d)